radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_FLYSKY_TELEMETRY            TR("FlySky RSSI #", "Use FlySky RSSI")		/* use english */
+#define TR_GPS_COORDS_FORMAT           TR("GPS Координати", "Формат координат")
+#define TR_VARIO                       TR("Варіо", "Варіометр")
+#define TR_PITCH_AT_ZERO               "Крок нуль"
+#define TR_PITCH_AT_MAX                "Крок max"
+#define TR_REPEAT_AT_ZERO              "Порвтори нуль"
+#define TR_BATT_CALIB                  TR("Batt. калібр.", "Battery калібровка")
+#define TR_CURRENT_CALIB               "Калібровка струму"
+#define TR_VOLTAGE                     TR(INDENT "Напруга", INDENT "Джерело напруги")
+#define TR_SELECT_MODEL                "Оберіть модель"
+#define TR_MANAGE_MODELS               "КЕРУВАННЯ МОДЕЛЯМИ"
+#define TR_MODELS                      "Моделі"
+#define TR_SELECT_MODE                 "Оберіть режим"
+#define TR_CREATE_MODEL                "Створіть модель"
+#define TR_FAVORITE_LABEL              "Обрані"
+#define TR_MODELS_MOVED                "Невикористані моделі переміщено до"
+#define TR_NEW_MODEL                   "Нова Модель"
+#define TR_INVALID_MODEL               "Невірна Модель"
+#define TR_EDIT_LABELS                 "Редагувати Мітки"
+#define TR_LABEL_MODEL                 "Мітка моделі"
+#define TR_MOVE_UP                     "Рух вгору"
+#define TR_MOVE_DOWN                   "Рух вниз"
+#define TR_ENTER_LABEL                 "Введіть мітку"
+#define TR_LABEL                       "Мітка"
+#define TR_LABELS                      "мітки"
+#define TR_CURRENT_MODEL               "Поточна"
+#define TR_ACTIVE                      "Активна"
+#define TR_NEW                         "Нова"
+#define TR_NEW_LABEL                   "Нова мітка"
+#define TR_RENAME_LABEL                "Перейменувати мітку"
+#define TR_DELETE_LABEL                "Видалити мітку"
+#define TR_DUPLICATE_MODEL             "Дублювати мітку"
+#define TR_COPY_MODEL                  "Копіювати модель"
+#define TR_MOVE_MODEL                  "Перемістити модель"
+#define TR_BACKUP_MODEL                "Резерв копія моделі"
+#define TR_DELETE_MODEL                "Видалити модель"
+#define TR_RESTORE_MODEL               "Відновити модель"
+#define TR_DELETE_ERROR                "Видалити помилку"
+#define TR_SDCARD_ERROR                TR("помилка SD", "помилка SD карти")
+#define TR_SDCARD                      "SD Карта"
+#define TR_NO_FILES_ON_SD              "Нема файлів на SD!"
+#define TR_NO_SDCARD                   "SD карта відсутня"
+#define TR_WAITING_FOR_RX              "Чекаємо на RX..."
+#define TR_WAITING_FOR_TX              "Чекаємо на TX..."
+#define TR_WAITING_FOR_MODULE          TR("Чекаємо на модуль", "Чекаємо на модуль...")
+#define TR_NO_TOOLS                    "Немає інструментів"
+#define TR_NORMAL                      "Нормально"
+#define TR_NOT_INVERTED                "Не інв."
+#define TR_NOT_CONNECTED               TR("!З'єдн.", "Не з'єднано");
```

Comment:
A more user-friendly version. Let's not expect from a user to know a Boolean algebra 😄 
```suggestion
#define TR_NOT_CONNECTED               TR("Не з'єдн.", "Не з'єднано");
```

Reply:
I generally agree, but in the context of OpenTX and EdgeTX, it's actually expected that the user would know boolean Algebra, as it's used in several places in the system.